app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -218,52 +222,54 @@ class LuciBuildService {
       }
     }
 
-    return builderNames;
+    return targets;
   }
 
-  Future<List<String>> _builderNamesForRepo(
-    List<LuciBuilder> builders,
-    github.PullRequest pullRequest,
-  ) async {
-    final Map<String?, Build?> builds = await tryBuildsForPullRequest(pullRequest);
-    if (builds.values.any((Build? build) {
-      return build?.status == Status.scheduled || build?.status == Status.started;
-    })) {
+  /// List of targets that are not running.
+  ///
+  /// Throws [InternalServerError] if any target is scheduled or running on [pullRequest].
+  Future<List<Target>> _targetsNotScheduled(List<Target> targets, github.PullRequest pullRequest) async {
```

Comment:
I'm not sure this is necessary (I didn't find any logs related to what this is catching), but I left it in to maintain parity

Reply:
Removed it per keyong's request